screens/ProfileScreen.js

```diff
@@ -32,7 +36,18 @@ const ProfileScreen = ({navigation}) => {
       );
     console.log('Sign_out clicked');
   };
-  /* 
+
+  const updateUsername = (newUsername) => {
+    console.log('ProfileScreen received this', newUsername);
+    auth()
```

Comment:
Should `return` here (return a `Promise`) to make the `updateUsername` chainable, waiting for it to be successful to run `setDetail` and `toggleDetailModal` in `DetailUpdateModal`